src/common/common.py

```
@@ -199,21 +209,27 @@ def page_setup(page: str = "") -> dict[str, Any]:
             os.chdir("../streamlit-template")
         # Define the directory where all workspaces will be stored
         workspaces_dir = Path("..", "workspaces-" + st.session_state.settings["repository-name"])
-        if "workspace" in st.query_params:
-            st.session_state.workspace = Path(workspaces_dir, st.query_params.workspace)
-        elif st.session_state.location == "online":
-            workspace_id = str(uuid.uuid1())
-            st.session_state.workspace = Path(workspaces_dir, workspace_id)
-            st.query_params.workspace = workspace_id
+        # Check if workspace logic exist if it is keep existing logic or set the default workspace but without params
+        if st.session_state.settings["is_workspace_enabled"] == True:
```

Ruff (0.8.2)

Avoid equality comparisons to `True`; use `if st.session_state.settings["is_workspace_enabled"]:` for truth checks

Replace with `st.session_state.settings["is_workspace_enabled"]`

(E712)